### ISSUE TITLE
augment on enums

### ISSUE THREAD

**Issue by gfldex** [perl6/roast#214]
RT will follow soon

**Comment by gfldex** [comment]
depends on https://rt.perl.org/Public/Bug/Display.html?id=130501

**Comment by b2gills** [comment]
I don't think this should be part of Perl 6, and I don't think it should necessarily be removed from Rakudo
